src/images/hex-import-background.svg

```diff
@@ -0,0 +1 @@
17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zM1428.5 1416.6a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zM1563.9 1416.6a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zM1703 1416.6a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zM1428.5 938.6a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zM1701 938.6a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zM1428.5 1177.3a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zM1563.9 1177.3a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zM1703 1177.3a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zM1494.4 1297.4a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zM1634.4 1297.4a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zM1770.7 1297.4a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zM1496.4 1058.1a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8V1008a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zM1360.7 1058.1a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8V1008a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zM1293 939.5a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zM1632.4 1058.1a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8V1008a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1z"/></g></g></svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1822 1440">
  <style>
    .import-bg{fill-rule:evenodd;clip-rule:evenodd;fill:#9b9b9b}
  </style>
  <g opacity=".15" transform="translate(14 43)">
      <path d="M135.5 816.6a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8L204 726.6a16.7 16.7 0 0 0-17 0l-43 25.1a17 17 0 0 0-8.5 14.8v50.1z" class="import-bg"/>
      <path d="M270.9 816.6a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm142.1-.2a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zM135.5 577.3a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8L204 487.3a16.7 16.7 0 0 0-17 0l-43 25.1a17 17 0 0 0-8.5 14.8v50.1zm135.4 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm139.1 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zM135.5 92.4a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8V42.2a17 17 0 0 0-8.5-14.8L204 2.3a16.7 16.7 0 0 0-17 0l-43 25.1a17 17 0 0 0-8.5 14.8v50.2zm135.4 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8V42.2a17 17 0 0 0-8.5-14.8L339.5 2.3a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.2zm139.1 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8V42.2a17 17 0 0 0-8.5-14.8L478.6 2.3a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.2zM135.5 338a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8L204 247.9a16.7 16.7 0 0 0-17 0L144 273a17 17 0 0 0-8.5 14.8V338zm135.4 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8L339.5 248a16.7 16.7 0 0 0-17 0l-43.1 25a17 17 0 0 0-8.5 14.8V338zm139.1 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8L478.6 248a16.7 16.7 0 0 0-17 0l-43.1 25a17 17 0 0 0-8.5 14.8V338zM201.4 696.9a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8L270 606.9a16.7 16.7 0 0 0-17 0L209.9 632a17 17 0 0 0-8.5 14.8v50.1zm140 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8L410 606.9a16.7 16.7 0 0 0-17 0L349.9 632a17 17 0 0 0-8.5 14.8v50.1zm-140-238.8a17 17 0 0 0 8.5 14.8L253 498a16.7 16.7 0 0 0 17 0l43-25a17 17 0 0 0 8.5-14.8V408a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm140 0a17 17 0 0 0 8.5 14.8L393 498a16.7 16.7 0 0 0 17 0l43-25a17 17 0 0 0 8.5-14.8V408a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm136.3 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8V408a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zM201.4 214.9a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8L270 124.9a16.7 16.7 0 0 0-17 0L209.9 150a17 17 0 0 0-8.5 14.8v50.1zm-133.7 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0L76.2 150a17 17 0 0 0-8.5 14.8v50.1zM0 93.3a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8V43.1a17 17 0 0 0-8.5-14.8L68.6 3.2a16.7 16.7 0 0 0-17 0L8.5 28.3A17 17 0 0 0 0 43.1v50.2zm341.4 121.6a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8L410 124.9a16.7 16.7 0 0 0-17 0L349.9 150a17 17 0 0 0-8.5 14.8v50.1zm337.1 601.7a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8L747 726.6a16.7 16.7 0 0 0-17 0l-43 25.1a17 17 0 0 0-8.5 14.8v50.1zm135.4 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zM678.5 577.3a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8L747 487.3a16.7 16.7 0 0 0-17 0l-43 25.1a17 17 0 0 0-8.5 14.8v50.1zm135.4 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zM953 816.6a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm0-239.3a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zM678.5 92.4a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8V42.2a17 17 0 0 0-8.5-14.8L747 2.3a16.7 16.7 0 0 0-17 0l-43 25.1a17 17 0 0 0-8.5 14.8v50.2zm274.5 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8V42.2a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.2zM678.5 338a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8L747 247.9a16.7 16.7 0 0 0-17 0L687 273a17 17 0 0 0-8.5 14.8V338zm135.4 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8L882.5 248a16.7 16.7 0 0 0-17 0l-43.1 25a17 17 0 0 0-8.5 14.8V338zm139.1 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25a17 17 0 0 0-8.5 14.8V338zm136 .4a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zM884.4 696.9a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8L953 606.9a16.7 16.7 0 0 0-17 0L892.9 632a17 17 0 0 0-8.5 14.8v50.1zm-140-238.8a17 17 0 0 0 8.5 14.8L796 498a16.7 16.7 0 0 0 17 0l43-25a17 17 0 0 0 8.5-14.8V408a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm-132.4.3a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm408.7-.3a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8V408a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zM744.4 215.9a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8L813 125.9a16.7 16.7 0 0 0-17 0L752.9 151a17 17 0 0 0-8.5 14.8v50.1zm-133.7 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0L619.2 151a17 17 0 0 0-8.5 14.8v50.1zM543 93.3a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8V43.1a17 17 0 0 0-8.5-14.8L611.6 3.2a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.2zm341.4 122.6a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8L953 125.9a16.7 16.7 0 0 0-17 0L892.9 151a17 17 0 0 0-8.5 14.8v50.1zm340.1 603.7a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm135.4 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm-135.4-242.3a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm135.4 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zM1499 819.6a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm0-242.3a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm138 .1a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm-138-485a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8V42.2a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.2zM1224.5 338a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25a17 17 0 0 0-8.5 14.8V338zm135.4 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8V338zm139.1 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8V338zm138 .4a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm139 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm-345.6 358.5a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm-410.4.5a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm138 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm132.4-239.3a17 17 0 0 0 8.5 14.8L1342 498a16.7 16.7 0 0 0 17 0l43-25a17 17 0 0 0 8.5-14.8V408a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm140 0a17 17 0 0 0 8.5 14.8L1482 498a16.7 16.7 0 0 0 17 0l43-25a17 17 0 0 0 8.5-14.8V408a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm136.3 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8V408a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm-276.3-244.2a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm-133.7 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zM1089 93.3a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8V43.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.2zm341.4 120.6a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zM338.5 1416.6a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm135.4 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm139.1 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm-274.5-478a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8L407 848.5a16.7 16.7 0 0 0-17 0l-43 25.1a17 17 0 0 0-8.5 14.8v50.2zm274.5 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm-274.5 238.7a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm135.4 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm139.1 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm135 .3a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm-343.6 119.8a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm140 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm136.3 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm-275.3-239.3a17 17 0 0 0 8.5 14.8L457 1098a16.7 16.7 0 0 0 17 0l43-25a17 17 0 0 0 8.5-14.8V1008a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm-134.7 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8V1008a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zM203 939.5a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm339.4 118.6a17 17 0 0 0 8.5 14.8L594 1098a16.7 16.7 0 0 0 17 0l43-25a17 17 0 0 0 8.5-14.8V1008a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm340.1 358.2a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm274.5 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zM882.5 938.4a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8L951 848.3a16.7 16.7 0 0 0-17 0l-43 25.1a17 17 0 0 0-8.5 14.8v50.2zm135.4 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm139.1 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zM882.5 1177a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25a17 17 0 0 0-8.5 14.8v50.2zm135.4 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm139.1 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm134 .4a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm-342.6 119.7a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8V1247a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm140 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8V1247a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm136.3 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8V1247a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm-276.3-239.2a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0L956.9 993a17 17 0 0 0-8.5 14.8v50.1zm-133.7 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0L823.2 993a17 17 0 0 0-8.5 14.8v50.1zM747 939.3a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm681.5 477.3a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm135.4 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm139.1 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm-274.5-478a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm272.5 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm-272.5 238.7a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm135.4 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm139.1 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm-208.6 120.1a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm140 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm136.3 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm-274.3-239.3a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8V1008a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm-135.7 0a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8V1008a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zM1293 939.5a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8v-50.1a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1zm339.4 118.6a17 17 0 0 0 8.5 14.8l43.1 25.1a16.7 16.7 0 0 0 17 0l43.1-25.1a17 17 0 0 0 8.5-14.8V1008a17 17 0 0 0-8.5-14.8l-43.1-25.1a16.7 16.7 0 0 0-17 0l-43.1 25.1a17 17 0 0 0-8.5 14.8v50.1z" class="import-bg"/>
  </g>
</svg>
```

Reply:
(Just getting rid of id's, making the class unique, and formatting for readability; the version for the live code will be minified automatically in the build process)

Reply:
Thanks :)